src/pages/iou/IOUTransactions.js

```diff
@@ -86,6 +86,7 @@ class IOUTransactions extends Component {
                         <ReportTransaction
                             chatReportID={this.props.chatReportID}
                             iouReportID={this.props.iouReportID}
+                            reportActions={this.props.reportActions}
```

Comment:
`reportActions` passed but not defined in the prop types?

Reply:
Actually I need to remove this as it's not needed. I had initially included a bug fix in this PR that should have its own issue, and forgot to remove this part. Thanks 👍🏻